app/models/user.rb

```diff
@@ -12,6 +12,7 @@ class User < ApplicationRecord
   has_many :technologies, through: :user_technologies
   validates :first_name, :last_name, presence:true
   validates :first_name, :last_name, length: { minimum: 2 }
-  # validates :personal_summary, length: { in: 50..600 }
+  validates :personal_summary, length: { in: 20..1200 }
```

Comment:
did we decide to go back to forcing the user to have their p_summary? if not, we can do without this validation, maybe?

Reply:
yeah sorry. forgot to comment that out again.
```suggestion
  #validates :personal_summary, length: { in: 20..1200 }
```